site/blog/announcements/open-source.mdx

```diff
@@ -21,7 +21,7 @@ We are excited to announce that [IaSQL](https://iasql.com) is now open source! T
 
 IaSQL also makes it possible to express infrastructure changes as code that can be version controlled. This can be done with any [migration system](https://en.wikipedia.org/wiki/Schema_migration) for schema and data changes, or in a script using [idempotent SQL inserts](https://www.prisma.io/dataguide/postgresql/inserting-and-modifying-data/insert-on-conflict) more akin to [IaC](https://en.wikipedia.org/wiki/Infrastructure_as_code).
 
-PostgreSQL IaSQL databases can be provisioned and configured via our [dashboard](https://app.iasql.com). The dashboard calls the [IaSQL container](https://github.com/iasql/iasql) which is a Node.js server written in Typescript that provisions unmodified PostgreSQL instances loaded with tables representing AWS services controlled via the [AWS SDK](https://www.npmjs.com/package/aws-sdk). AWS is our main focus at the moment, but we plan to support GCP, Azure and other cloud providers soon. This is an [updated list](https://github.com/iasql/iasql#cloud-providers) of the AWS services that we currently support. Let us know if you need a specific AWS service and we should be able prioritize it!
+PostgreSQL IaSQL databases can be provisioned and configured locally or via our hosted offering. The dashboard calls the [IaSQL container](https://github.com/iasql/iasql) which is a Node.js server written in Typescript that provisions unmodified PostgreSQL instances loaded with tables representing AWS services controlled via the [AWS SDK](https://www.npmjs.com/package/aws-sdk). AWS is our main focus at the moment, but we plan to support GCP, Azure and other cloud providers soon. This is an [updated list](https://github.com/iasql/iasql#cloud-providers) of the AWS services that we currently support. Let us know if you need a specific AWS service and we should be able prioritize it!
```

Comment:
maybe add the link for the hosted offering to app.iasql.com?

Reply:
since this post is dated from last year I would rather not include it in case something changes down the line and the link breaks unexpectedly